Lanchonete/Lanchonete/ Features/Customization/Presentation/CustomizationViewPresenter.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  CustomizationViewPresenter.swift
+//  Lanchonete
+//
+//  Created by vera.dias on 2/9/21.
+//
+
+import Foundation
+
+public protocol CustomizationView: class {
+    func reloadData()
+}
+public class CustomizationViewPresenter {
+    let ingredientManager: IngredientManager
+    let allIngredientManager = AllIngredientsManager()
```

Comment:
`AllIngredientsManager` should probably also be injected in the constructor.

Reply:
Added tests also